whichone/static/js/form.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function () {
```

Comment:
Looks like this is not required? As I can see the DOM has href on the anchor element.

Reply:
removed